graphene_django/views.py

```diff
@@ -280,8 +280,11 @@ def can_display_graphiql(cls, request, data):
     @classmethod
     def request_wants_html(cls, request):
         accepted = get_accepted_content_types(request)
-        html_index = accepted.count('text/html')
-        json_index = accepted.count('application/json')
+        accepted_length = len(accepted)
+        #the list will be ordered in preferred first - so we have to make
+        #sure the most preferred gets the highest number
+        html_index = accepted_length - accepted.index('text/html') if 'text/html' in accepted else 0
```

Comment:
Yeah, `count` is definitely not accounting for priority here. Thanks for the fix.

Can we rename `html_index` and `json_index` to `html_priority` and `json_priority`? I was a little confused at first because their index wouldn't need `accepted_length`. But I now see why we need the `accepted_length - accepted.index(...` part to make the `html_priority > json_priority` condition work.

Thanks!

Reply:
Good Point - the names were leftovers from before the fixes - I have updated the  names